src/main/java/frc/robot/controls/Driver.java

```diff
@@ -39,8 +39,8 @@
 
 	// CONSTANTS //
     public static final double DEADZONE = 0.15; // Deadzone applied to joysticks to aid in adjusting sensitivity
-    public static final double THROTTLE_LOW_SPEED = 0.5;
-    public static final double STEER_LOW_SPEED = 0.4;
+    public static final double THROTTLE_LOW_SPEED = 1;
+    public static final double STEER_LOW_SPEED = 0.85;
     public static final double THROTTLE_HIGH_SPEED = .8;
     public static final double STEER_HIGH_SPEED = .75;
```

Comment:
Why is low speed higher than high speed?

Reply:
Originally it was supposed to be lower but it turned out that we never needed to move slower than high speed. Then when I needed to go even faster I hijacked the variables I already had. I can change the name of all the variables and methods after provincials, but I don't want to change it right now because it's not major and I might accidentally break stuff.